releasenotes/notes/deprecated-positional-arguments-cb78dd22eb0894e0.yaml

```diff
@@ -0,0 +1,8 @@
+---
+issues:
+  - |
+    If positional arguments were passed into QSVR or QSVC and these classes were printed, an 	 exception was raised.
+fixes:
+  - |
+    Positional arguments in QSVR and QSVC are deprecated.
```

Comment:
This should be a `deprecations` section since it notes what is deprecated really not any fixes.